wagtailio/static/sass/components/_error-404.scss

```diff
@@ -0,0 +1,48 @@
+.error-404 {
+    position: relative;
+
+    &__intro,
+    &__heading {
```

Comment:
Question: Is it better to use two classes that do the same thing in this case or one?

Reply:
`__intro` requires [some additional styles](https://github.com/wagtail/wagtail.org/blob/50-error-pages/wagtailio/static/sass/components/_error-404.scss#L17) which is why they're different. I also like that `__heading` clearly refers to a `<h1>` tag and intro a `<p>`.

 I guess you could have a new class for grid alignment, which thinking about it we have a ticket for in the backlog for - to have grid helper classes as at the moment we have some repetition of aligning items using `grid-column: x / span x`